### PR TITLE
Fix charts not showing up on home page, and hide chart titles

### LINKED ISSUE #1686: Blog post teaser charts not displaying on homepage for featured blog posts
When featured on the homepage, blog posts with a teaser graphic chart set don't get the chart rendered:

Image

I suspect we need to add the rendering javascript to the homepage. It should render the same as it does on a blog index page:

Image

Though in both places I think we should also not render the chart title (it makes the graphic too tall and is unnecessary with the article title there).

### PR DESCRIPTION
**Pull request by dkaoster**
closes #1686 1686

Fixes teaser image on posts not showing up in the home page, and fixes titles showing up on these post lists.

### CODE REVIEW

File: client/charts/js/bubble-map-chart.js [hunk 1]
```diff
@@ -46,3 +46,17 @@ function engageCharts() {
 }
 
 engageCharts();
+
```
Comment (dkaoster) [inline client/charts/js/bubble-map-chart.js:49]: Realized that I forgot this in the previous PR, adding this in here

File: common/models/pages.py [hunk 2]
```diff
@@ -163,6 +163,14 @@ def streamfield_media(self, media_type):
         except Exception:
             pass
 
+        # If the page is a home page, loop through all the panels and posts to see if
```
Comment (dkaoster) [inline common/models/pages.py:166]: This makes sure the js is included on the home page.

File: common/templates/common/blocks/bubble_map_chart.html [hunk 3]
```diff
@@ -1,4 +1,4 @@
-{% if value.title %}<h2>{{ value.title }}</h2>{% endif %}
+{% if static_graphic is None and value.title %}<h2>{{ value.title }}</h2>{% endif %}
```
Comment (dkaoster) [inline common/templates/common/blocks/bubble_map_chart.html:1]: This hides the title if the graphic is a "static_graphic" when used in blog listing page or as primary image on the top of a page.

**Comment by SaptakS** [comment]
When I tried to use bar chart as teaser image, it shows up like this 
image

Is this how it is supposed to show? I feel like there should be black borders somewhere. Though maybe this is a separate issue requiring a separate PR.

**Comment by SaptakS** [comment]
One other thing I noticed is that graphic takes some time to load for obvious reason. Can we have a placeholder till the vizualization is loaded to avoid any jump in the layout?

**Comment by harrislapiroff** [comment]
Looks like in [the figma](https://www.figma.com/file/H3uTXxT88ofhdebUFgzEo9/PFT-%E2%80%93-Hand-off-Guide?type=design&node-id=3689-55821&mode=design&t=jM0YYpGBK8qOE8U8-4) they add a subtle darker background. I don't know how big of a difference that makes, but we can open a separate ticket for this and for the loading time, but I'd like to get this fix merged if it's otherwise good. Did it look good to you otherwise @SaptakS?

image

**Review by harrislapiroff — APPROVED** [review]
Saptak confirmed in slack that this PR looks good otherwise, so I'm going to merge and open new tickets for the other issues.

**Pull request merged**